res/.htaccess

```diff
@@ -15,3 +15,31 @@ RewriteRule ^ - [L]
 
 # If the requested resource doesn't exist, use index.html
 RewriteRule ^ /index.html
+
+# Security Headers
+# IMPORTANT NOTE: KEEP THEM SYNCED WITH THE DEV SERVER (see server.js)
+
+# Do not try to guess the content-type for JS and CSS files served with a wrong mime-type.
+Header set X-Content-Type-Options nosniff
+# This is specific to Safari, Chrome and Internet Explorer, to provide some XSS
+# Protection for versions that do not support CSP yet.
+Header set X-XSS-Protection "1; mode=block"
+# Do not allow being embedded in a frame.
+Header set X-Frame-Options SAMEORIGIN
```

Comment:
Hmm, this feels a bit hostile to me. There is a hypothetical use case of people wanting to embed perf.html on a different site, for example to show multiple profiles in a long scrollable page. This header would make that impossible.

However, I can also see an argument for keeping this header. Once we store profiles locally, an attacker might trick a user into uploading a private profile unknowingly, through clickjacking. So let's err on the side of security / privacy and keep this header.

Reply:
We should re-visit this decision once we turn this into a DevTools product.

Reply:
Yeah; for now I decided to go the strictest we could do, but we can relax some if needed :)